fpmsyncd/routesync.cpp

```diff
@@ -25,20 +30,41 @@ RouteSync::RouteSync(RedisPipeline *pipeline) :
 void RouteSync::onMsg(int nlmsg_type, struct nl_object *obj)
 {
     struct rtnl_route *route_obj = (struct rtnl_route *)obj;
-    struct nl_addr *dip;
-    char destipprefix[MAX_ADDR_SIZE + 1] = {0};
-
-    dip = rtnl_route_get_dst(route_obj);
-    nl_addr2str(dip, destipprefix, MAX_ADDR_SIZE);
-    SWSS_LOG_DEBUG("Receive new route message dest ip prefix: %s\n", destipprefix);
+    
     /* Supports IPv4 or IPv6 address, otherwise return immediately */
     auto family = rtnl_route_get_family(route_obj);
     if (family != AF_INET && family != AF_INET6)
     {
-        SWSS_LOG_INFO("Unknown route family support: %s (object: %s)\n", destipprefix, nl_object_get_type(obj));
+        SWSS_LOG_INFO("Unknown route family support (object: %s)\n", nl_object_get_type(obj));
         return;
     }
 
+    /* Get the index of routing table */
+    unsigned int table_index = rtnl_route_get_table(route_obj);
+    
+    /* Default routing table. This line may have problems. */
+    if (table_index == RT_TABLE_UNSPEC) 
```

Comment:
This could be unsafe to classify every other routes to be VNET. As we discussed, it would be safe if we can use the id range check for table index

Reply:
Will address this in the future